shard/management/commands/get_shard_command.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand
+from pprint import pprint
+
+from shard.utils.shard import get_shard_by_shard_key_and_shard_group
+
+
+class Command(BaseCommand):
+    title = 'Get shard by key and group'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--shard_group', '-s',
+            type=str,
+            dest='shard_group',
+            help='Shard group',
+        )
+
+        parser.add_argument(
+            '--shard_key', '-k',
+            type=str,
+            dest='shard key',
+            help='Shard Key',
+        )
+
+    @staticmethod
+    def assert_if_params_are_not_valid(**options):
```

Comment:
This method is unnecessary.

`argparse` module include `required` option. 
If `required` is True and parameter is null, can't passing validation.

You don't have to write additional code.